lms/envs/test.py

```diff
@@ -339,6 +339,9 @@
     ("utm_medium", "Footer"),
 ])
 
+MIDDLEWARE.remove('openedx.features.edly.middleware.EdlyOrganizationAccessMiddleware')
+MIDDLEWARE.remove('openedx.features.edly.middleware.SettingsOverrideMiddleware')
```

Comment:
I think the edX test should run fine with`SettingsOverrideMiddleware`. If this is the case then please remove it from here.

Reply:
some tests had issues with it. its better to not use this with the tests that we haven't written i think